jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -833,6 +823,26 @@ properties:
             type: number
             default: 0.0
             fits_keyword: V3_REF
+          vparity:
+            title: V2-V3 coordinate frame parity
+            type: integer
+            default: 1.0
+            fits_keyword: VPARITY
+          v3yangle:
+            title: V3 position angle (deg)
+            type: number
+            default: 0.0
+            fits_keyword: V3I_YANG
```

Comment:
PA_V3 is one of the keyword values calculated and populated by Perry's pointing script. So it's an output rather than an input.